packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -560,8 +560,8 @@ export const messages = {
   conflict_detect_diff_command_title: 'Compare Files',
 
   force_source_diff_text: 'SFDX: Diff File Against Org',
-  force_source_diff_remote_not_found:
-    'Selected file or directory was not found in the org',
+  force_source_diff_components_not_in_org:
```

Comment:
FYI @sbudhirajadoc

Reply:
We have already reviewed this one for directory diff PR. This PR is removing the duplicate messages.